data-model/SDP-data-model-templates/place_sdp-template.json

```diff
@@ -13,9 +23,26 @@
       "uniform_name": ""
     }
   ],
-  "associated_entities": [
+  "related_persons": [
+    {
+      "relation": "CV",
+      "id": "URI/ARK",
+      "note": [
+        "array of strings"
+      ]
+    }
+  ],
+  "related_places": [
```

Comment:
This could include 'within' relations (e.g., settlement to region/district/country)